core/src/main/java/org/jruby/RubyFile.java

```diff
@@ -418,24 +418,21 @@ public IRubyObject ctime(ThreadContext context) {
     public IRubyObject birthtime(ThreadContext context) {
         checkClosed(context);
 
-        FileTime btime = getBirthtimeWithNIO(getPath());
-        if (btime != null) return context.runtime.newTime(btime.toMillis()); // btime comes in nanos
-        return ctime(context);
-    }
-
-    public static final FileTime getBirthtimeWithNIO(String pathString) {
```

Comment:
Mild concern here about removing a public method, but I don't know how long it has been around.